translations/translate.R

```diff
@@ -165,6 +165,8 @@ jaspXgettext2pot <-
 #  Start of copied code from R-SVN File src/library/tools/R/translations.R;
 #  Refactoring update_pkg_po -> jasp_update_pkg_po.
 jasp_update_pkg_po <- function(pkgdir, pkg = NULL, version = NULL,
+                          pot_make = TRUE, mo_make = TRUE,
+                          verbose = getOption("verbose"),
```

Comment:
Maybe this isn't necessary because this is just code copied from upstream, and the first commit is to catch upstream's changes. The really valid commit is the second one.

Reply:
The real problem is not here, but that tools::update_pkg_po in baseR does not assume in advance that all pot files are UTF-8, which is confirmed by the GNU gettext tool they quoted. I just manually hardcoded it to UTF-8, see main comment details.